mlrun/utils/notifications/notification_pusher.py

```diff
@@ -122,7 +122,9 @@ def should_send_notification(self, notification_type):
 
         # get notification's inverse dependencies, and only send the notification if
         # none of its inverse dependencies are being sent
-        inverse_dependencies = notification_type.inverse_dependencies()
+        inverse_dependencies = NotificationTypes(
+            notification_type
+        ).inverse_dependencies()
         for inverse_dependency in inverse_dependencies:
```

Comment:
@quaark, before, the type of `notification_type` was `str`, now its the correct one.